lib/codit/outbreak.py

```diff
@@ -1,11 +1,30 @@
 import pandas as pd
 import logging
+import numpy as np
+import math
+import matplotlib.pyplot as plt
+# according to Leeds accommodations' coordinates distribution's 'lon':'lat' ratio, the unit is inch, real size would fit the distribution
+plt.rcParams["figure.figsize"] = [12, 5]
+# Need "pip install celluloid", celluloid -- a 3rd-party python lib using matplotlib.animation.ArtistAnimation
+from celluloid import Camera
 
 from codit.population.covid import PersonCovid
 from codit.population.population import FixedNetworkPopulation
 
 from codit.disease import covid_hazard
 
+def setup_range_for_histogram(pop):
```

Comment:
Could this be moved to a module of its own? Could you encapsulate much of this as an object, say the OutbreakVisualiser()?

Reply:
Yes, will do.